docs_source_files/content/remote_webdriver/remote_webdriver_client.ko.md

```diff
@@ -54,7 +54,17 @@ driver.quit()
 To further customize our test configuration, we can add other desired capabilities.
 
 
+<<<<<<< HEAD
```

Comment:
Could you please solve this merge conflict? Seems you commited the diff instead of solving it.
Basically, lines 57 to 65 should be deleted. Also line 67.

Reply:
Sorry for my mistake. Deleted that part 😉